app/main.py

```diff
@@ -1,8 +1,42 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class, clean_mark, brand):
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center, clean_power,
+                 average_rating, count_of_ratings):
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars_list):
+        result = 0
+        for car in cars_list:
+            if self.calculate_washing_price(car) is not None:
+                result += self.calculate_washing_price(car)
+                self.wash_single_car(car)
+        return round(result, 1)
+
+    def calculate_washing_price(self, car):
+        if car.clean_mark < self.clean_power:
+            difference = self.clean_power - car.clean_mark
+            income = \
+                car.comfort_class * difference * self.average_rating / \
+                self.distance_from_city_center
```

Comment:
If you have some long math, you can split it into additional variables.

Good example:

```python
fuel_consumption = max_fuel_consumption * height_fuel_consumption_coeficient
estimated_speed = plan_max_speed - wind_awerage_speed * wind_angle_coefisient
estimated_time = distance_to_the_destinatoin / estimated_speed
how_much_fuel_needed = fuel_consumption * estimated_time * overlap_coeficient
```

Bad example:

```python
how_much_fuel_needed = max_fuel_consumption \
                       * height_fuel_consumption_coeficient \
                       * distance_to_the_destinatoin / (
                               plan_max_speed 
                               - wind_awerage_speed 
                               * wind_angle_coefisient
                       ) * overlap_coeficient
```